VirtualList.svelte

```diff
@@ -33,6 +29,12 @@
 	$: if (mounted) refresh(items, viewport_height, itemHeight);
 
 	async function refresh(items, viewport_height, itemHeight) {
+		const isStartOverflow = items.length < start
+		
+		if (isStartOverflow) {
+			await _scrollTo(items.length - 1, {behavior: 'auto'})
```

Comment:
wrong function name `_scrollTo` need to `scrollToIndex` from line 114

Reply:
thx for the review. I'll have a look :slightly_smiling_face:

Reply:
yes was pretty obvious. Guess I cannot code anymore without eslint telling me, I've got unused functions after renaming them :sweat_smile: 

I've updated the code as recommended :+1: